wp-content/themes/haruoshi_origin/src/scss/archive-news.scss

```
@@ -3,15 +3,22 @@
 
 .news {
   &__article {
-    border-bottom: solid 1px $accent-color;
-    padding: 15px;
     display: flex;
     align-items: center;
+    @include mq-sp {
+      display: block;
+      text-align: left;
+    }
+    border-bottom: solid 1px $accent-color;
+    padding: 15px;
```

```suggestion
    border-bottom: solid 1px $accent-color;
    padding: 15px;
    display: flex;
    align-items: center;
    @include mq-sp {
      display: block;
      text-align: left;
    }
```
スタイル指定の途中にメディアクエリ書くのはあんまりしないかな。